db/migrate/20181009090135_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :admin, :bool, default: false
```

Comment:
this should be `:boolean` https://api.rubyonrails.org/classes/ActiveRecord/Migration.html